src/transformers/tokenization_utils_base.py

```diff
@@ -2448,23 +2486,40 @@ def save_pretrained(
             tokenizer_config["extra_special_tokens"] = self.extra_special_tokens
             tokenizer_config.update(self.extra_special_tokens)
 
-        saved_raw_chat_template = False
-        if self.chat_template is not None:
-            if isinstance(self.chat_template, dict):
-                # Chat template dicts are saved to the config as lists of dicts with fixed key names.
-                # They will be reconstructed as a single dict during loading.
-                # We're trying to discourage chat template dicts, and they are always
-                # saved in the config, never as single files.
-                tokenizer_config["chat_template"] = [{"name": k, "template": v} for k, v in self.chat_template.items()]
-            elif kwargs.get("save_raw_chat_template", False):
-                with open(chat_template_file, "w", encoding="utf-8") as f:
-                    f.write(self.chat_template)
-                saved_raw_chat_template = True
-                logger.info(f"chat template saved in {chat_template_file}")
-                if "chat_template" in tokenizer_config:
-                    tokenizer_config.pop("chat_template")  # To ensure it doesn't somehow end up in the config too
-            else:
-                tokenizer_config["chat_template"] = self.chat_template
+        saved_raw_chat_template_files = []
+        if kwargs.get("save_jinja_files", False) and isinstance(self.chat_template, str):
```

Comment:
```suggestion
        if kwargs.get("save_jinja_files", True) and isinstance(self.chat_template, str):
```
same

Reply:
@julien-c Sure, if you want! I was going to leave the default save format as the old style for one or two versions so that people don't get errors when they load models in the new format, but if you're happy to change immediately, I can do it!

Reply:
let's change immediately imo !